core-java-modules/core-java-dates-operations/src/test/java/com/baeldung/date/comparison/DateComparisonUtilsUnitTest.java

```diff
@@ -0,0 +1,47 @@
+package com.baeldung.date.comparison;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.Date;
+
+import org.junit.jupiter.api.Test;
+
+public class DateComparisonUtilsUnitTest {
+
+	private Date toDate(LocalDateTime localDateTime) {
+		return Date.from(localDateTime.atZone(ZoneId.systemDefault()).toInstant());
+	}
+
+	Date day1Morning = toDate(LocalDateTime.of(2019, 10, 19, 6, 30, 40));

```

Comment:
Can we make these date variables private as I don't think they are referenced anywhere else? Also, let's move them up to the top of the class definition.